ci-operator/config/operator-framework/community-operators/operator-framework-community-operators-master.yaml

```diff
@@ -0,0 +1,24 @@
+base_images:
+  operator-test-playbooks:
+    name: ansible-runner
+    namespace: openshift
+    tag: latest
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.5"
+  namespace: ocp
+tests:
+- as: ansible
+  commands: test-commands.sh
+  container:
+    from: src
```

Comment:
@petr-muller, we thought that `src` is pointer to base_image. Should we use `operator-test-playbooks` instead of `src`?

Reply:
I don't know what `operator-test-playbooks` is, so I have no idea, You'd probably want to use the image that has `test-commands.sh` (if that lives in `operator-framework/community-operators` then `src` or something derived from it is likely a good candidate) together with all dependencies (usually supplied in the build root).